<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelWindow</class>
 <widget class="QMainWindow" name="PanelWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>831</width>
      <height>461</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="info_Widget" native="true">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>60</y>
       <width>521</width>
       <height>361</height>
      </rect>
     </property>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>151</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="finalize_PushButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>320</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="infoSearch_LineEdit">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>181</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="infoDisplay1_Lable">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="infoDisplay2_Lable">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="infoDisplay1_LineEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="infoDisplay2_LineEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>191</width>
        <height>24</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QScrollArea" name="infoSearchResults_ScrollArea">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>181</width>
        <height>271</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>179</width>
         <height>269</height>
        </rect>
       </property>
       <widget class="QFrame" name="frame_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>181</width>
          <height>271</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="search_Widget" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>181</width>
       <height>411</height>
      </rect>
     </property>
     <widget class="QCheckBox" name="searchAll_CheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>78</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Show All</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="searchBooks_CheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Books</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="searchPeople_CheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Customers</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="search_LineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Search something...</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="searchResults_ScrollArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>181</width>
        <height>271</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>179</width>
         <height>269</height>
        </rect>
       </property>
       <widget class="QFrame" name="frame_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>181</width>
          <height>271</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="buttons_Widget" native="true">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>521</width>
       <height>41</height>
      </rect>
     </property>
     <widget class="QPushButton" name="addBook_PushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Book</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addCustomer_PushButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Customer</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
